src/scripts.js

```diff
@@ -110,21 +121,72 @@ window.addEventListener('click', closeMessage)
 
 
 // ------EVENT HANDLERS/FUNCTIONS------
-function setUpCustomerDashboard() {
-  getCumulativeCost();
+function checkEmptyInputs(event) {
+  event.preventDefault()
+  if(!usernameInput.value || !passwordInput.value) {
+    hide(unmatchedCredentialsError)
+    show(emptyFieldsError)
+  } else {
+    checkValidInputs()
+  }
+}
+
+function checkValidInputs() {
+  const usernameID = parseUsername(usernameInput.value)
+  const isValidPassword = parsePassword(passwordInput.value)
+  if (usernameID && isValidPassword) {
+    hide(emptyFieldsError)
+    hide(unmatchedCredentialsError)
+    show(loadingCircle)
+    setTimeout(() => {
+      setUpCustomerDashboard(usernameID)
+      hide(loadingCircle)
+      }, 2000)
+    
+  } else {
+    hide(emptyFieldsError)
+    show(unmatchedCredentialsError)
+  }
+}
+
+
+function parseUsername(username) {
+  const foundUser = store.customerRepo.allCustomers.find((customer) => {
+    return customer.username === username
+  })
+  if(foundUser) {
+    return foundUser.id
+  }
+}
+
+function parsePassword(password) {
+  if(password === 'overlook2021') {
+    return true
+  } else {
+    return false
+  }
```

Comment:
One way to DRY this up is do on `line 163` return `password === 'overlook2021' and remove lines `164-166`. When you are returning just `true` or `false` you can just return what you are evaluating.